book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] As the `attr_accessor` creates both the READER & WRITER methods. I would recommend including the `rentals` in the attr_accessor too.

Reply:
Thanks for the feeback 
i have resolved it